utils/fixed_decimal/src/decimal.rs

```diff
@@ -2701,6 +2725,113 @@ fn test_pad() {
     assert_eq!("-00.42", dec.to_string());
 }
 
+#[test]
+fn test_sign_display() {
```

Comment:
Question: Is this a pre-existing test that you converted, or did you write it fresh?

Reply:
I wrote it fresh.